README.md

```diff
@@ -1,15 +1,22 @@
 # Ploigos GitHub Workflows
 
-In order to use this workflow for your application, you must do the following -
+This repository houses the Ploigos workflows defined within a Github Actions [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows). For ease of use, there are templates that can be used within the examples directory. Please replace all values that begin with `REPLACE_`.
 
-Create a psr.yaml file in your application repository
+## Setup Instructions
 
-Example - ./example/psr.yaml
+1. Create a psr.yaml file in your application repository and supply all needed values.
+   * `cp ./examples/psr.yaml <application repository>/.`
```

Comment:
Instead of maintaining an examples folder, let's reference 1) the spring-petclinic example application and 2) the example application-ops repo.

For every item we have tagged with "REPLACE_", we should add a note to the readme of that repository that "To re-use this you have to replace the following values"

Reply:
this is done